lib/application/widgets/text_tag.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/widgets.dart';
+import 'package:memo/application/constants/dimensions.dart' as dimens;
+
+/// Creates a new [Text] that resembles a tag
+///
+/// It's important to no confuse this tag-styled component with the material's `Chip`, which is used in more complex
+/// - usually interactable - scenarios.
+class TextTag extends StatelessWidget {
+  const TextTag(
+    this.text, {
+    required this.backgroundColor,
+    required this.padding,
```

Comment:
I couldn't understand the use-case that needs this `padding` to be parametrized in the Widget